cmd/snap/cmd_list.go

```diff
@@ -124,6 +130,30 @@ func listSnaps(names []string, all bool) error {
 	w := tabWriter()
 	defer w.Flush()
 
+	switch format {
+	case "":
+		return outputSnapsDefault(w, snaps)
+	default:
+		return outputSnapsWithFormat(Stdout, snaps, format)
+	}
+}
+
+func outputSnapsWithFormat(w io.Writer, snaps []*client.Snap, format string) error {
+	t, err := template.New("list-output").Parse(format)
+	if err != nil {
+		return fmt.Errorf("cannot use given template: %q", err)
```

Comment:
This will never teach people how to use this feature properly. Can we perhaps hint at the format spec (ideally in the man page but our manual pages don't have too much space to add some free form text that would give an example)